app/models/site_customization/content_block.rb

```diff
@@ -1,5 +1,5 @@
 class SiteCustomization::ContentBlock < ActiveRecord::Base
-  VALID_BLOCKS = %w(top_links footer)
+  VALID_BLOCKS = %w(top_links footer subnavigation_left subnavigation_right)
```

Comment:
@decabeza I made some changes in relation with the keys used for the content block. They are now named as you suggested (subnavigation_left, subnavigation_right).

Reply:
👌